src/pages/announcements/Announcements.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { FunctionComponent, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { AppBar, Button, Grid, List, Toolbar } from '@mui/material';
+import { AddCircleOutline } from '@mui/icons-material';
+import { CreateAnnouncementDialog } from './CreateAnnouncementDialog';
+import { useAnnouncements } from './useAnnouncements';
+import { ListItemAnnouncement } from './ListItemAnnouncement';
+
+const Announcements: FunctionComponent = () => {
```

Comment:
Maybe we can separate in different components the top bar from the announcements list